src/apps/trainings/models/network.py

```diff
@@ -128,6 +128,7 @@ class Meta:
     log_gamma_upper_confidence = FloatField(
         _("log gamma upper confidence"), default=0, db_index=True, help_text=_("Upper confidence bound on BayesElo strength of network."),
     )
+    log_gamma_game_count = BigIntegerField(_("log gamma game count"), default=0, db_index=True, help_text=_("Number of real games used to compute log_gamma for this network."),)
```

Comment:
You prefixed it with log_gamma because it can be different from the actual game count ? (eg new game came but elo wasn't updated ?)

Reply:
Yes, this specifically is the game count that is matching with the Elo computed.